ouroboros-consensus/src/Ouroboros/Storage/VolatileDB/Impl.hs

```diff
@@ -418,10 +429,11 @@ tryCollectFile :: forall m h blockId
                -> InternalState blockId h
                -> (FileId, FileInfo blockId)
                -> m (InternalState blockId h)
-tryCollectFile hasFS@HasFS{..} env slot st@InternalState{..} (fileId, fileInfo) =
+tryCollectFile hasFS env@VolatileDBEnv{..} slot st@InternalState{..}
+  (fileId, fileInfo) =
```

Comment:
I would revert this change (so it's one line again) and use the following instead:

```haskell
tryCollectFile ...
    ....
  where
    VolatileDBEnv { _dbErr, _tracer } = env
```

Reply:
This is not so easy because `VolatileDBEnv` is Existential type, but I did the same for `InternalState` to keep it a single like.